web/static/ux/css-reset.js

```diff
@@ -0,0 +1,21 @@
+import { css } from 'https://cdn.jsdelivr.net/gh/lit/dist@3/all/lit-all.min.js';
+
+export default css `
+    *, *::before, *::after {
+        box-sizing: border-box;
+    }
+
+    * {
+        margin: 0;
+        padding: 0;
+    }
+
+    ul, ol {
+        list-style: none;
+        padding: 0;
+    }
+
+    a {
+        text-decoration: none;
```

Comment:
I could do without obfuscating all links.

Reply:
It should work the same as the one previously defined in [main.css](https://github.com/filecoin-project/curio/blob/3e37077a14f42f8b075576f37ed668cee31e6df3/web/static/ux/main.css#L76), though I agree that some default link styling would be nice, I was thinking about adjusting it in the next revamps.